src/main/java/com/sports/server/command/timeline/domain/QuarterChangeTimeline.java

```diff
@@ -0,0 +1,16 @@
+package com.sports.server.command.timeline.domain;
+
+import jakarta.persistence.DiscriminatorValue;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import lombok.Getter;
+
+@Entity
+@DiscriminatorValue("QUARTER_CHANGE")
+@Getter
+public class QuarterChangeTimeline extends Timeline  {
+
+    @Enumerated(EnumType.STRING)
+    private QuarterChangeType quarterChangeType;
```

Comment:
``@Column``이 빠진게 아닐까 싶어

```suggestion
    @Enumerated(EnumType.STRING)
    @Column(name = "quarter_change_type")
    private QuarterChangeType quarterChangeType;
```